test_folder/test_file.txt

```diff
@@ -0,0 +1 @@
+Test 2 and 3
```

Comment:
@prvbharadwaj1731 These are files you should keep out of your repo, either by using a `test_folder` outside your repo or adding to `.gitignore`.